.github/workflows/ci.yml

```diff
@@ -15,8 +15,10 @@ jobs:
     strategy:
       matrix:
         command:
-          - php:sniff
```

Comment:
In `composer.json` we have the order
- `lint`
- `sniff`
- `fixer`
- `md`
- `psalm`

I thought this was already the most logical order?  Though it may be a moot point as the actions run in parallel.

Reply:
The order in the `composer.json` was somewhat arbitrary/random/organically-grown. The order in the `.travis.yml` followed a rough "more functional checks before more cosmetic checks, and faster checks before slower checks" order. For GitHub actions (which indeed get executed almost in parallel), I have used an alphabetical order.

Reply:
> I have used an alphabetical order.

OK, fair enough.